lib/ims/lti/services/tool_consumer_profile_service.rb

```diff
@@ -0,0 +1,16 @@
+module IMS::LTI::Services
+  class ToolConsumerProfileService
+
+    attr_accessor :tcp
+
+    def initialize(tool_consumer_profile)
+      @tcp = tool_consumer_profile
+    end
+
+    def supports_capabilities?(capability, *capabilities)
```

Comment:
Is there a downside of changing the signature to 
```
supports_capabilities?( *capabilities)
```
and dropping `capabilities.unshift(capability)` from line 11?